README.md

```diff
@@ -173,31 +173,30 @@ Now make some changes to your source code. Then:
 - After making Python code changes, restarting the notebook kernel will be enough to reflect changes
 - After making JavaScript code changes:
     - `cd js`
-    - `npm run build`
+    - `yarn run build`
```

Comment:
Should the readme instructions mention installing `yarn`? (since it's not installed by default with node like npm)